indexer/src/types/transaction.ts

```diff
@@ -28,35 +26,69 @@ import {
   TxValuesArray,
   TypedTransaction,
   TypedTxData,
+  hexToBytes,
+  bytesToBigInt,
+  JsonTx,
 } from "../deps.ts";
 
+type HexString = `0x${string}`
+
+interface ToEthTxRequest {
```

Comment:
Can we create a separate dedicated file where we store all the interfaces and types as they are constants and can be reused for other purposes?

Reply:
No problem, should I create a file like "interfaces.ts" or keep all the interfaces in the "dep.ts" file with the other interfaces?

Reply:
Created a file named "interfaces.ts" in the same folder. If this doesn't match what is needed, please specify the correct location for the file.